src/content/components/FlameChartViewport.js

```diff
@@ -1,9 +1,26 @@
-import React, { Component, PropTypes } from 'react';
+// @flow
+import React, { Component } from 'react';
 import FlameChartCanvas from './FlameChartCanvas';
+import type { Thread } from '../../common/types/profile';
+import type { Milliseconds, CssPixels, UnitIntervalOfProfileRange } from '../../common/types/units';
+import type { StackTimingByDepth } from '../stack-timing';
+
+type Props = {
+  connectedProps: {
```

Comment:
It was me not wanting to enumerate a bunch of things in the original code since we're mostly passing properties from the connected component down through the viewport to the canvas component. I guess it's kind of silly now that I spelled it out like this.